adapted-articles/K/KingdomofGodKingdomofHeaven.xml

```diff
@@ -2,62 +2,62 @@
+<p>When <span class="hebrew">malkut</span> is used of God, it almost always refers to his authority or to his rule as the heavenly King. “They will talk together about the glory of your kingdom; they will celebrate examples of your power. . . . For your kingdom is an everlasting kingdom. You rule generation after generation” (<a href="?bref=Ps.145.11,13">Ps 145:11, 13</a>).</p>
+<p>Further, if a king rules, there must be a realm or sphere over which he reigns. This is also called <span class="hebrew">malkut</span><span class="ital">.</span> “So the realm of Jehoshaphat was quiet, for his God gave him rest round about” (<a href="?bref=2Chr.20.30">2 Chr 20:30</a>; see <a href="?bref=Esth.3.6">Est 3:6</a>; <a href="?bref=Jer.10.7">Jer 10:7</a>; <a href="?bref=Dan.9.1">Dn 9:1</a>; <a href="?bref=Dan.11.9">11:9</a>).</p>
+<p>This same twofold use of <span class="greek">basileia</span> is found in the NT. In fact, <span class="greek">basileia</span> could be translated by the expression “kingly power” in <a href="?bref=Luke.23.42">Luke 23:42</a> and by “kingship” in <a href="?bref=John.18.36">John 18:36</a>. When a nobleman went into a far country to get a “kingdom” (<a href="?bref=Luke.19.12">Luke 19:12</a>) he went to the governing authority to get an appointment as king. When Jesus said, “My kingship is not of this world” (<a href="?bref=John.18.36">Jn 18:36</a>), he did not mean to say that his rule has nothing to do with the world but rather that his kingship—his dominion—does not come from man but from God. Therefore, he rejects the use of worldly fighting to gain his ends.</p>
+<p>This central meaning of <span class="greek">basileia</span> makes it easy to understand many sayings in the Gospels. In the Lord’s Prayer the petition “Your kingdom come” (<a href="?bref=Matt.6.10">Mt 6:10</a>) is a prayer for God to manifest his reign so that his will may be done on earth as it is in heaven. When we read that we are to “receive the kingdom of God like a child” (<a href="?bref=Mark.10.15">Mk 10:15</a>), we must open our hearts and lives to the rule of God.</p>
+<p>Also in the NT are sayings about being <span class="ital">in</span> the kingdom or of <span class="ital">entering</span> the kingdom (<a href="?bref=Matt.8.11">Mt 8:11</a>; <a href="?bref=Mark.9.47">Mk 9:47</a>; <a href="?bref=Mark.10.23-25">10:23-25</a>; <a href="?bref=Luke.13.28">Lk 13:28</a>). From the perspectives of language development (philology) and theology, there is no reason not to understand “the kingdom of God” first as the divine reign or rule and second as the sphere of blessing in which that reign is experienced.</p>
```

Comment:
The double negative is hard to translate and also pointless. "We may understand" or "It is entirely reasonable to understand"

Reply:
The first suggestion is better plain English, but I prefer the second as it’s a little closer in meaning to the original.